### DISCUSSION TITLE
Why I always get a warning that Ocamlformat_rpc is not installed?

### DISCUSSION THREAD

**Discussion by anentropic** [ocamllabs/vscode-ocaml-platform#868] [Q&A] [answered]
Every time I start VS Code with my ocaml project open I get a notification warning:

![Screenshot 2022-02-14 at 19 28 39](https://user-images.githubusercontent.com/147840/153932770-f8428845-4909-430c-be9b-905d9a7cd503.png)

But `opam list` in the VS Code terminal shows:
```
ocamlformat           0.19.0      Auto-formatter for OCaml code
ocamlformat-rpc-lib   0.19.0      Auto-formatter for OCaml code (RPC mode)
```

and everything seems to work ok (but maybe I'm silently missing out on some auto formatting?)

How do I make it happy?

**Comment by ulugbekna** [comment]
You should install the package ocamlformat-rpc. It should be mentioned in this repo’s readme btw

**Comment by anentropic** [comment]
Install it where?  I thought I already had it installed - see above

**Comment by mlantas** [comment, marked as answer]
You only have `ocamlformat-rpc-lib` installed. You need to install `ocamlformat-rpc`.

**Comment by anentropic** [comment]
Ah, thanks!  It seems obvious, but I wasn't seeing it 🤦

**Reply by rgrinberg** [reply]
The error message isn't exactly obvious. We should hint at the correct package in that message.

**Reply by rgrinberg** [reply]
I improved the error message in ocamllsp#master.

**Reply by anentropic** [reply]
thanks 👍

another notification I just encountered in another project:
![Screenshot 2022-02-15 at 17 03 47](https://user-images.githubusercontent.com/147840/154112017-9e470ad5-c9c8-4732-aa26-1d9be8615846.png)

I think this message could be improved in a similar way - I believe the opam package that should be upgraded is called `ocaml-lsp-server` rather than `ocamllsp`?
